src/pudl/package_data/phmsagas/column_maps/yearly_transmission_failures_leaks_repairs.csv

```diff
@@ -1,18 +1,19 @@
 year_index,1990,1991,1992,1993,1994,1995,1996,1997,1998,1999,2000,2001,2002,2003,2004,2005,2006,2007,2008,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020,2021,2022
-data_date,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of
+data_date,,,,,,,,,,,,,,,,,,,,,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of,datafile_as_of
 report_year,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,yr,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year,report_year
+filing_date,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,doe,,,,,,,,,,,,,
```

Comment:
We don't keep `filing_date` in the other tables other than A-D, I believe. But I also see an argument for keeping it. Let's merge this for now and take this up in #3277, as it'll affect all the tables we've integrated up until now.

Reply:
hm yea good point! but agreed let's punt it. I'll add a quick comment over there about all of the date fields.